src/edu/ucsd/library/xdre/collection/CollectionHandler.java

```diff
@@ -1172,7 +1172,15 @@ public static String objectPart(String symbol){
 	public static String nextBlankNodeId(){
 		return "_" + (""+random.nextInt()).replace("-", "n");
 	}
-	
+
+	public static boolean isDerivativesRequired(String fid, String use) {
+		if ( StringUtils.isBlank(fid))
+			return false;
+		return (isImage(fid, use) || isDocument(fid, use) || isVideo(fid, use) || isAudio(fid, use))
+				&& ((use == null && fid.startsWith("1.")) 
```

Comment:
The boolean conditionals here are pretty nested. Adding to it/modifying it, might be complicated if changes are needed in the future. Could we perhaps break up the checking into two separate conditionals?

1. Is the file an image, document, video or audio file?
2. Is the file a source file or a service file being used as a source file (1.xxx)?